configure

```diff
@@ -225,33 +237,38 @@ parser.add_option('--with-etw',
     dest='with_etw',
     help='build with ETW (default is true on Windows)')
 
-parser.add_option('--download',
+parser.add_option('--with-intl',
     action='store',
-    dest='download_list',
-    help=nodedownload.help())
+    dest='with_intl',
+    default='none',
+    choices=valid_intl_modes,
+    help='Intl mode (valid choices: {0}) [default: %default]'.format(
+        ', '.join(valid_intl_modes)))
 
-parser.add_option('--with-icu-path',
+intl_optgroup.add_option('--with-icu-path',
     action='store',
     dest='with_icu_path',
     help='Path to icu.gyp (ICU i18n, Chromium version only.)')
 
-parser.add_option('--with-icu-locales',
+intl_optgroup.add_option('--with-icu-locales',
     action='store',
     dest='with_icu_locales',
     default='root,en',
     help='Comma-separated list of locales for "small-icu". "root" is assumed. '
         '[default: %default]')
 
-parser.add_option('--with-intl',
-    action='store',
-    dest='with_intl',
-    help='Intl mode: none, full-icu, small-icu [default: none]')
-
-parser.add_option('--with-icu-source',
+intl_optgroup.add_option('--with-icu-source',
     action='store',
     dest='with_icu_source',
     help='Intl mode: optional local path to icu/ dir, or path/URL of icu source archive.')
 
+intl_optgroup.add_option('--download',
```

Comment:
Judging by the push-back I'd say that we may not be adding anything else and an objective would be to remove the one we have now. Although I'm personally in favour of _download all the things_.